macros/parsers/parserPopUp.pl

```diff
@@ -221,8 +224,9 @@ sub MENU {
 			$menu .= $escaped_item . '</li>' . "\n";
 		}
 		$menu .= '</var>';
-	} elsif ($main::displayMode eq "TeX") {
-		return unless $self->{static};
+	} elsif ($main::displayMode eq 'PTX' && !$self->{showInStatic}) {
+		$menu = qq(<fillin name="$name"/>);
```

Comment:
I don't really like the approach of
```
if ($main::displayMode eq 'PTX' && $self->{showInStatic}) {
} elsif ($main::displayMode eq 'PTX' && !$self->{showInStatic}) {
}
```
that you are employing to avoid nesting if statements.
In my opinion it would be better to do
```
if ($main::displayMode eq 'PTX') {
    if ($self->showInStatic) {
    } else {
    }
}
```
I understand that you don't like the nested `if`s, but it is a natural language construct, and doesn't involve unnecessary conditional checking.  Indentation makes the nesting clear to me.

This is just my opinion, so no need to change it.

Reply:
And I changed it to nest the ifs here.